apps/qubit/modules/search/templates/_aggregation.php

```diff
@@ -26,7 +26,7 @@
         <li <?php if ($active) echo 'class="active"' ?>>
           <?php echo link_to(__(render_value_inline($bucket['display'])) . '<span>, ' . $bucket['doc_count'] . ' ' . __('results') . '</span>',
             array('page' => null, $name => $bucket['key'] == 'unique_language' ? null : $bucket['key']) +
-            $sf_data->getRaw('sf_request')->getParameterHolder()->getAll(), array('title' => '')) ?>
+            $sf_data->getRaw('sf_request')->getParameterHolder()->getAll(), array('title' => __(render_value_inline($bucket['display'])))) ?>
```

Comment:
I think it's probably better to strip any markdown and just use the plain text value, e.g. `'title' => __(strip_markdown($bucket['display'])`.  I'm not sure if it's best to strip markdown from the link value on line 28 too, e.g. `link_to(__(strip_markdown($bucket['display']))`?

Reply:
Didn't know we had such a function! Yeah, seems like doing it on line 28 makes too. Updated!